cloud/sshclient/sshclient.go

```diff
@@ -2,19 +2,26 @@ package sshclient
 
 import (
 	"errors"
+	"fmt"
 	"os"
 	"os/exec"
 	"strconv"
+
+	"go.jetpack.io/devbox/debug"
 )
 
 type Client struct {
-	Username string
-	Hostname string
-	Port     int
+	Username       string
+	Hostname       string
+	Port           int
+	ProjectDirName string
 }
 
 func (c *Client) Shell() error {
-	cmd := c.cmd()
+	cmd := c.cmd("-t")
+	remoteCmd := fmt.Sprintf("bash -l -c \"start_devbox_shell.sh %s 300\"", c.ProjectDirName)
```

Comment:
The project name needs quotes in case of spaces:

```suggestion
	remoteCmd := fmt.Sprintf(`bash -l -c "start_devbox_shell.sh \"%s\" 300"`, c.ProjectDirName)
```

Reply:
good catch!